ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/messages/LimitedTransactionsMessagesTest.java

```diff
@@ -56,7 +57,9 @@ public void createLimited() {
     List.of(firstMessage, secondMessage).stream()
         .map(message -> message.getTransactionsMessage().getSize())
         .forEach(
-            messageSize -> assertThat(messageSize).isLessThan(LimitedTransactionsMessages.LIMIT));
+            messageSize ->
+                assertThat(messageSize)
+                    .isLessThan(LimitedTransactionsMessages.LIMIT + MAX_ADDITIONAL_BYTES));
```

Comment:
Why can't we stay under the `LIMIT` anymore?

Reply:
We can get over the limit because we are checking the length of the message against LimitedTransactionsMessages.LIMIT before we are adding the length of that RLP encoded list. 
Looking at commit messages it looks like this is a self imposed limit. It changes from limiting the number of transactions to limiting the size. I guess that means that it should not matter if there are 5 bytes more. I will slack Abdel to make sure I'm right.